From c42d23a7f9f902de2eefcac0c23094ad4021cb94 Mon Sep 17 00:00:00 2001
From: David Bauer <mail@david-bauer.net>
Date: Wed, 26 Nov 2025 13:34:48 +0100
Subject: [PATCH] ftrace

---
 targets/generic | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/targets/generic b/targets/generic
index 21f07162..95b8f232 100644
--- a/targets/generic
+++ b/targets/generic
@@ -62,6 +62,13 @@ try_config('TARGET_SQUASHFS_BLOCK_SIZE', 256)
 -- (e.g. ramips-mt76x8) by default.
 config('DROPBEAR_ED25519', true)
 
+config('KERNEL_FTRACE', true)
+config('KERNEL_FTRACE_SYSCALLS', true)
+config('KERNEL_ENABLE_DEFAULT_TRACERS', true)
+config('KERNEL_FUNCTION_TRACER', true)
+config('KERNEL_FUNCTION_GRAPH_TRACER', true)
+config('KERNEL_DYNAMIC_FTRACE', true)
+config('KERNEL_ENABLE_DEFAULT_TRACERS', true)
 config('KERNEL_PROC_STRIPPED', true)
 config('KERNEL_AIO', false)
 try_config('KERNEL_IO_URING', false)
@@ -129,4 +136,5 @@ packages {
 	'-firewall4',
 	'gluon-core',
 	'ip6tables-zz-legacy',
+	'trace-cmd'
 }
-- 
2.51.0

